modules/claims_api/app/controllers/claims_api/v0/claims_controller.rb

```diff
@@ -33,26 +32,26 @@ def show # rubocop:disable Metrics/MethodLength
           # NOTE: source doesn't seem to be accessible within a remote evss_claim
           render json: evss_claim, serializer: ClaimsApi::ClaimDetailSerializer
         else
-          render json: { errors: [{ status: 404, detail: 'Claim not found' }] },
-                 status: :not_found
+          raise ::Common::Exceptions::ResourceNotFound.new(detail: 'Claim not found')
         end
       rescue => e
         log_message_to_sentry('Error in claims show',
                               :warning,
                               body: e.message)
-        render json: { errors: [{ status: 404, detail: 'Claim not found' }] },
-               status: :not_found
+        raise if e.is_a?(::Common::Exceptions::UnprocessableEntity)
```

Comment:
cant you just rescue ::Common::Exceptions::UnprocessableEntity and do a raise
and then also rescue a StandardError (after the above rescue) and raise claim not found?

this works too, but if what i described works, you wouldnt have to check what class the exception is explicitly like this